src/main/java/org/jenkinsci/plugins/ssegateway/sse/EventDispatcher.java

```diff
@@ -212,7 +212,7 @@ public boolean unsubscribe(@Nonnull EventFilter filter) {
                 );
                 return true;
             } else {
-                LOGGER.log(Level.WARNING, "Invalid SSE unsubscribe configuration. No active subscription matching filter: ");
+                LOGGER.log(Level.FINE, "Invalid SSE unsubscribe configuration. No active subscription matching filter: ");
```

Comment:
Changing WARNING to FINE is good idea 👍 However the message itself looks odd, perhaps it should be:

```
LOGGER.log(Level.FINE, "Invalid SSE unsubscribe configuration. No active subscription for channel: " + channelName));
```